src/bentoml_cli/containerize.py

```diff
@@ -540,14 +540,27 @@ def containerize(  # type: ignore
                         "To load image built with 'buildctl' requires one of docker, podman, nerdctl (Neither are found in PATH) and try again."
                     )
                     sys.exit(0)
-                o = subprocess.check_output([container_runtime, "load"], input=result)
-                click.echo(o.decode("utf-8").strip())
+                # NOTE: We will always use the docker image spec if docker is available.
+                # Otherwise fallback to the OCI image spec.
+                if "output" not in _memoized:
```

Comment:
What's this change do?

Reply:
deprecation warning right?. I also add the instruction for loading the tar into local container registry.

Reply:
I'm not sure why we need to warn twice if you haven't specified output?

Reply:
We need to do that right. First warning is to say the autoconfig is deprecated. The second one is instruct how they can do it with passing in output.